webpack/assets/javascripts/react_app/components/FactCharts/constants.js

```diff
@@ -0,0 +1 @@
+export const FACT_CHART = 'FACT_CHART';
```

Comment:
can we move it into the `slice.js` file as it's part of the redux implementation?